import "server_data.proto";
import "auto_data.proto";

message StartServerInitReq
{
	optional string name = 1;
	optional int32 type = 2;
}

message StartServerInitRes
{
	optional string config = 1;
}

message EndServerInitNotice
{
	optional string name = 1;
}

message SyncServerNotice
{
	repeated Server servers = 1;
}

message HeartBeatNotice
{
	optional string name = 1;
}

message ProtocolWrapper
{
	optional int32 protocol_id = 1;
	optional ClientConnInfo client_conn_info = 2;
	optional bytes inner_protocol = 3;
}